modules/openapi-generator/src/main/resources/typescript-angular/README.mustache

```diff
@@ -201,3 +201,21 @@ import { environment } from '../environments/environment';
 })
 export class AppModule { }
 ```
+
+### Customizing parameter encoding
+
+Without further customization, only parameters of style 'simple' are encoded correctly.
+
+Other styles ("matrix", "label", "form", ...) are not that easy to encode and thus are best delegated to
```

Comment:
can you link the openapi spec, e.g. https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#parameter-locations?